examples/atlas-streams/README.md

```diff
@@ -0,0 +1,24 @@
+# MongoDB Atlas Provider - Atlas Streams with Terraform
```

Comment:
@Zuhairahmed Let me know if there is anything else you would like to add in this section.